scale: 0.04
relative: 20.0
# KF covariance matrices for all static objects (unknown static)
Q: [0.0125,0.0125,1.0,0.001,0.001]
R: [0.0001,0.0005,0.025]
# KF covariance matrices for dynamic object (pedestrian and unknown dynamic)
Q_dynamic: [0.015,0.015,1.0,0.05,0.05]
R_dynamic: [0.0015,0.0015,0.05]
# new_value = old_value + Kalman_gain * (detection - old_value)
one_d_kalman_gains: [0.50, 0.40, 0.42, 0.75, 0.3]   # Kw, Kl, Kh, Kyaw, Kconfidence
confidence_drop: 0.0    # How much should confidence drop over time c(t) = c(0) * exp(-1 * confidence_drop * t)
metricGate: 0.9
ioa_thres: 0.8
delete_time: 9999.0        # How many seconds to wait before deleting an object with no associated detections
metric_thres: 0.5       # metric distance
pedestrian_type: 1      # This type corresponds to a "pedestrian" object
unknown_type: 3         # This type corresponds to an "unknown" object
unknown_dynamic_type: 4 # This type corresponds to an "unknown dynamic" object
# Scaling factors for cost function parameters: d = d_euclid + alpha * d_size + beta * d_confidence
cost_alpha: 1000.0
cost_beta: 0.5
# HiddenMarkovModel Parameters
types: [0, 1, 2]           # [Pedestrian, Unknown Static, Unknown Dynamic]
# p(z_t = i | z_{t-1} = j)
A_hmm: [0.75, 0.10, 0.10,        # Transition Matrix (Columns must sum to one)
        0.15, 0.75, 0.15,
        0.10, 0.15, 0.75]
# p(x_t = l | z_t = k)
C_hmm: [0.80, 0.10, 0.10,       # Observation Matrix (Columns must sum to one)
        0.10, 0.80, 0.10,
        0.10, 0.10, 0.80]
# initial p(z_0) guess
pi_hmm: [0.325, 0.35, 0.325]
# static/dynamic object confidences
dynamic_obj_conf: 0.95
static_obj_conf: 0.725
